addon/styles/addon.scss

```diff
@@ -1,35 +1,38 @@
 @import 'node_modules/ember-frost-core/addon/styles/frost-theme';
+$infobarHeight: 80px;
+$infobarBorder: 1px;
```

Comment:
We've been using kebab-case for SASS variables so I'd change these variable names to be:
```sass
$info-bar-height: 80px;
$info-bar-border-width: 1px;
```